rocketpool/watchtower/submit-rewards-tree-rolling.go

```diff
@@ -506,23 +504,14 @@ func (t *submitRewardsTree_Rolling) runRewardsIntervalReport(client *rocketpool.
 
 		t.log.Printlnf("%s Merkle rewards tree for interval %d already exists at %s, attempting to resubmit...", t.logPrefix, currentIndex, rewardsTreePath)
 
-		// Try to upload the file
-		cid, err := t.uploadFileToWeb3Storage(fileBytes, compressedRewardsTreePath, "compressed rewards tree")
+		cid, err := rprewards.SingleFileDirIPFSCid(fileBytes, rewardsTreePath, "compressed rewards tree")
 		if err != nil {
-			// If it fails, get the CID and continue the submission process
-			t.log.Printf("error uploading Merkle tree to Web3.Storage: %s. Calculating CID", err.Error())
-			c, err := rprewards.GetCIDForSerializedFile(fileBytes, filepath.Base(compressedRewardsTreePath))
-			if err != nil {
-				return fmt.Errorf("error getting CID for file %s: %w", compressedRewardsTreePath, err)
-			}
-			cid = c.String()
-			t.printMessage(fmt.Sprintf("Calculated rewards file CID %s", cid))
-		} else {
-			t.log.Printlnf("%s Uploaded Merkle tree with CID %s", t.logPrefix, cid)
+			return fmt.Errorf("error submitting rewards snapshot: %w", err)
```

Comment:
reword this comment to specify there was an error while calculating the CID.

"submitting rewards snapshot" is vague anyway- doesn't specify whether it's submitted to web3storage vs the odao contracts

edit: elsewhere you use `fmt.Errorf("Error getting CID for file %s: %w", compressedRewardsTreePath, err)`